### ISSUE TITLE
getEvolutionGraph not available in On-Premise installation

### ISSUE THREAD

**Issue by coemans-com** [Chardonneaur/ChatGPT#6]
The ChatGPT plugin can't return any insight because 'getEvolutionGraph' is not available. I installed the plugin on Matomo 5.1.1.

Error message:
The given JSON data does not contain any significant metrics to provide insights from. However, it indicates an error notification. The error message signifies that a method named 'getEvolutionGraph' does not exist or is not available in the specified module - '\Piwik\Plugins\VisitsSummary\API'. Thus, the key insights are that your software is experiencing an error due to missing or inaccessible method in the specific module, which needs to be addressed for normal functioning.

**Comment by ronan-hello** [comment]
Yes, unfortunately, all reports methods are not standard and require a specific development for evolution graph, this is in the backlog for next release.

**Comment by ronan-hello** [comment]
This is fixed in release 5.4.0, thanks for opening this issue
